noir/noir-repo/aztec_macros/src/lib.rs

```diff
@@ -84,7 +99,10 @@ fn transform_module(module: &mut SortedModule) -> Result<bool, AztecMacroError>
         if !check_for_storage_implementation(module, &storage_struct_name) {
             generate_storage_implementation(module, &storage_struct_name)?;
         }
-        generate_storage_layout(module, storage_struct_name)?;
+        // Make sure we're only generating the storage layout for the root crate
```

Comment:
Is this because contracts can now import other contracts' source code and we need to tell the compiler to not generate the importee's storage layout?

If so, it might be worth mentioning contracts-importing-contracts in this comment?

Reply:
Done!